packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -228,6 +237,25 @@ export interface ISummaryBaseConfiguration {
      * allowed before forcibly electing a new summarizer client.
      */
     maxOpsSinceLastSummary: number;
+
+    /**
+     * Defines the compression algorithm which should be used for compression of summaries.
+     */
+    compressionAlgorithm?: SummaryCompressionAlgorithms;
+
+    /**
+     * The threshold value of the minimum number of bytes the summary blob must have
+     * to apply the compression.
+     */
+    minSizeToCompress?: number;
+
+    /**
+     * True enables the Base64 encoding of the compressed byte array. The default is false,
+     * because it adds additional processing to the blob content. At present, probably due
+     * to the bug, the upload of the blob which is not Base64 encoded fails.
+     * TODO: Cleanup and possibly remove this entry if the issue is fixed.
+     */
+    isUseB64OnCompressed?: boolean;
```

Comment:
If you haven't already, could you create a GH issue w/info about what breaks?  (callstack, etc.)

Reply:
> If you haven't already, could you create a GH issue w/info about what breaks? (callstack, etc.)

The issue has already been created at #12509